pkg/apis/rbac/v1alpha1/types.go

```diff
@@ -48,10 +47,6 @@ const (
 type PolicyRule struct {
 	// Verbs is a list of Verbs that apply to ALL the ResourceKinds and AttributeRestrictions contained in this rule.  VerbAll represents all kinds.
 	Verbs []string `json:"verbs" protobuf:"bytes,1,rep,name=verbs"`
-	// AttributeRestrictions will vary depending on what the Authorizer/AuthorizationAttributeBuilder pair supports.
```

Comment:
as long as v1alpha1 is around, need to tombstone this protobuf index so we don't reuse it